readthedocs/projects/views/private.py

```diff
@@ -80,9 +81,24 @@ class ProjectDashboard(PrivateViewMixin, ListView):
     model = Project
     template_name = 'projects/project_dashboard.html'
 
+    def email_notification(self, request):
+        email_qs = request.user.emailaddress_set.filter(primary=True)
+        email = email_qs.first()
+        if email:
+            if not email.verified:
+                email_confirm_notification(user=request.user, success=False)
+        else:
+            email_confirm_notification(user=request.user, success=False)
+
     def get_queryset(self):
         return Project.objects.dashboard(self.request.user)
 
+    def get(self, request, *args, **kwargs):
+        super(ProjectDashboard, self).get(request, *args, **kwargs)
+        context = self.get_context_data()
+        self.email_notification(request)
+        return self.render_to_response(context)
```

Comment:
I think that you can rewrite this as,

    self.validate_primary_email(user)
    return super()

Note that I changed the function's name and the arguments that it receives plus relying on `super` instead of our code.